data/WI/incentives.json

```diff
@@ -1327,7 +1304,7 @@
       "homeowner"
     ],
     "short_description": {
-      "en": "Up to $300 rebate for Heat Pump Water Heaters. Must have UEF of at least 2.20."
+      "en": "Up to $300 rebate for heat pump water heaters. Must have UEF of at least 2.20."
```

Comment:
(major nitpick) it would be nice to standardize whether we refer to appliances as singular or plural, i.e. "Up to $300 rebate for heat pump water heaters" (in this incentive) vs "Up to $300 rebate for heat pump water heater" (in the one above). I think plural sounds better. I know that we're all over the place with this so definitely not your fault in this PR but while we're updating descriptions might as well standardize for WI!

Reply:
Great idea, will do!